doc/antora/antora.yml

```diff
@@ -11,7 +11,7 @@ start_page: ROOT:index.adoc
 nav:
 - modules/ROOT/nav.adoc
 - modules/reference/nav.adoc
-- modules/raddb/nav.adoc
+- modules/reference/raddb/nav.adoc
```

Comment:
Just drop this comment, and part of the next one where you comment this line out.  Its best to just delete the line.